### ISSUE TITLE
Backpropogation fails for large canvas sizes for `pydiffvg.Path`

### ISSUE THREAD

**Issue by midhunharikumar** [BachiLi/diffvg#24]
Values for canvas size > 32 fails at `loss.backward()` step running on GPU ( V100 ).

Settings:

canvas_width = 240
canvas_height = 240
num_paths = 10
num_segments = 10

Notebook freezes and `nvidia-smi` shows gpu memory usage drop to 0% . IPython kernel is reset.
